edraak.mk

```diff
@@ -0,0 +1,14 @@
+edraak.build.all:
+	$(MAKE) edraak.build.programs
+
+edraak.build.programs:
+	docker build -t edraak-devstack-programs -f ../edraak-programs/Dockerfile ../edraak-programs
+
+edraak.programs.install-all:
+	../edraak-programs/docker-exec.sh install_all
```

Comment:
It's hard to remember to run this command.

```suggestion
```

Please include the install command inside the `provision.sh` script -- somewhere near the lines below:

https://github.com/Edraak/devstack/blob/9fbebb178a9dd24e64194133d51bb9fbee75d01c/provision.sh#L37-L47

Reply:
This command become optional. Now we have `make edraak.provision` that triggers `./provision-edraak.sh'
Knowing that edx original provision is calling `./provision-edraak.sh' too